src/app/static/src/tests/components/runWorkflow/runWorkflow.test.ts

```diff
@@ -127,6 +128,9 @@ describe(`runWorkflow`, () => {
     })
 
     it(`can start and cancel workflow wizard correctly when starting a workflow wizard from clone`, async (done) => {
+        const mockSetReportsSource = jest.fn();
+        session.setSelectedWorkflowReportSource = mockSetReportsSource;
```

Comment:
I understand the need to add session here but I thought we usually have session tests in separate a file.

Reply:
Yes, the session function is being tested separately [here](https://github.com/vimc/orderly-web/pull/399/files#diff-30e150fb459817cf0f0093747b1bd14a6779809ad01f102f2cd248c6a975935eR58-R62), this is just the test that it is called from the component.